jobmaster/dm/checkpoint/agent.go

```diff
@@ -130,13 +134,19 @@ func (c *AgentImpl) IsFresh(ctx context.Context, workerType lib.WorkerType, task
 	return isSyncFresh(ctx, task.Cfg, db)
 }
 
+func createMetaDatabase(ctx context.Context, cfg *config.JobCfg, db *conn.BaseDB) error {
+	query := fmt.Sprintf("CREATE DATABASE IF NOT EXISTS %s", cfg.MetaSchema)
```

Comment:
should backquote it. we can use tiflow/pkg/quotes.QuoteName or dbutil like below